cli/state.rs

```diff
@@ -33,6 +33,13 @@ use std::str;
 use std::thread::JoinHandle;
 use std::time::Instant;
 
+#[derive(Copy, Clone, Eq, PartialEq)]
+pub enum DebugType {
+  Main, // Can be debugged, will wait for debugger when --inspect-brk given.
+  Dependent, // Can be debugged, never waits for debugger.
+  Internal, // No inspector instance is created.
```

Comment:
It would be better to use triple slash comments so that they show up in rustdoc.

Reply:
s/Internal/None/ ?

Reply:
These names don't seem to correspond to how they're described in the docstrings...

Why not something like this?

```rust
enum DebugType {
  /// Can be debugged, will wait for debugger when --inspect-brk given.
  Break,
  /// Can be debugged, never waits for debugger.
  NeverWait,
  /// No inspector instance is created.
  None,
}
```